.github/workflows/Image-build-Order-Management.yaml

```
@@ -0,0 +1,109 @@
+            echo "WORKFLOW_FILE=solutions-qa.yaml" >> $GITHUB_ENV
+          elif [ "${{ github.ref }}" == "refs/heads/release/0.5" ]; then
+            echo "WORKFLOW_FILE=solutions-dpg-dev.yaml" >> $GITHUB_ENV
+          fi
+
+      - name: Trigger Deployment Pipeline
+        run: |
+          curl -X POST \
+          -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: token ${{ secrets.GHUB_TOKEN }}" \
+          https://api.github.com/repos/pucardotorg/pucar-Devops/actions/workflows/${{ env.WORKFLOW_FILE }}/dispatches \
+          -d '{"ref":"Solutions-Pipeline"}'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)